client/src/helpers/APICalls/kanban/colors.ts

```diff
@@ -0,0 +1,30 @@
+import { ICardTag } from '../../../context/types/kanban';
+/**
+ * Colors for the CardForm
+ */
+export const CardTagColors: Record<string, ICardTag> = {
+  white: {
+    name: 'white',
+    hexCode: '#FFFFFF',
+  },
+  green: {
+    name: 'green',
+    hexCode: '#5ACD76',
+  },
+  red: {
+    name: 'red',
+    hexCode: '#FF5D48',
+  },
+  yellow: {
+    name: 'yellow',
+    hexCode: '#EDAB1D',
+  },
+  blue: {
+    name: 'blue',
+    hexCode: '#59B0FF',
+  },
+  purple: {
+    name: 'purple',
+    hexCode: '#D460F7',
+  },
+};
```

Comment:
Just to mention, I want to move this into the Theme file but I am not sure if it's possible to this this kind of implementation without extending the current PalatteOptions. Nonetheless, this was created for the functionality aspect of the Board.